### PR TITLE
add character images + basic targeting

### PR DESCRIPTION
**Pull request by mick-warehime**

### CODE REVIEW

**Comment by mick-warehime** [comment]
fixes #89

File: src/characters/character_base.py [hunk 1]
```diff
@@ -9,10 +9,24 @@
                                Stateful)
 
 
+class Position(object):
+
```
Comment (dvirk) [inline src/characters/character_base.py]: Is this in-game position or position for blitting on the screen?

Comment (dvirk) [reply src/characters/character_base.py]: After reviewing the code it looks like Position and imagepath are both used only to blit the character on the screen. Do we need this to actually be stored in the Character? Maybe we should have a separate object whose responsibiltiy is to blit the character on the screen.

Comment (mick-warehime) [reply src/characters/character_base.py]: agreed this was quick and dirty to stand up visual targeting. i created issue #112 to track refactoring the image class. i added your comments there. really only the combat view needs to care about the position of the enemies/players so it should move there.

**Pull request merged**